docs/FAQ.md

```diff
@@ -157,12 +157,22 @@ Usually, the different commits associated with the conflicted branch should all
 appear in the log, but if they don't you can use `jj branch list`to show all the
 commits associated with it.
 
+### How do I integrate Jujutsu with Gerrit? 
+
+At the moment you'll need a script, which adds the required fields for Gerrit
+like the `Change-Id` footer. Then `jj` can invoke it via an `$EDITOR` override
+in an aliased command. Here's an [example][gerrit-integration] from an 
+contributor (look for the `jj signoff` alias).
+
+We hope to integrate with Gerrit natively in the future.
+
```

Comment:
Maybe just add one extra sentence that says something like:

```
After you have attached the `Change-Id:` to the commit series, you'll have to manually `git push` your `HEAD` into the remote Gerrit branch `refs/for/$BRANCH`, where `$BRANCH` is the base branch you want your changes to go to.
```

That at least gives the basic idea without us having to commit too much support/documentation (yet)

Reply:
done.